addon/utils/group-utils.js

```diff
@@ -71,6 +71,50 @@ export function optionAtIndex(originalCollection, index) {
   })(originalCollection, false) || { disabled: false, option: undefined };
 }
 
+function copyGroup(group, suboptions) {
+  let groupCopy = { groupName: group.groupName, options: suboptions };
+  if (group.hasOwnProperty('disabled')) {
+    groupCopy.disabled = group.disabled;
+  }
+  return groupCopy;
+}
+
+export function filterOptionsWithOffset(options, text, matcher, offset, skipDisabled = false) {
```

Comment:
Perhaps this function could be unified with the regular filter? Not needed, just wondering.

Reply:
I removed the function and added `findOptionWithOffset`.